doc/quickstart/actinia_quickstart.rst

```diff
@@ -146,64 +66,10 @@ actinia server and credentials:
    export ACTINIA_PASSWORD='gu3st!pa55w0rd'
    export ACTINIA_URL='https://actinia.mundialis.de/latest'
 
-Notes on the usage of external data sources
```

Comment:
Removing this section to try and reduce length. Does the document still stand without it?

Reply:
In my view the import of external data sources is a cool feature since most data will not be locally available.

Reply:
In other words keep it.

Reply:
Okay, it's back in.